avocado/utils/data_structures.py

```diff
@@ -33,6 +34,24 @@ def ordered_list_unique(object_list):
     return [x for x in object_list if not (x in seen or seen_add(x))]
 
 
+def geometric_mean(values):
+    """
+    Evaluates the geometric mean for a list of numeric values.
```

Comment:
Maybe it's worth to mention, that this is the "slow" implementation of geometric mean, which works for high number of items. (there is faster implementation out there, which fails in that case).